R/zzz.R

```diff
@@ -14,15 +14,15 @@ initialize_options <- function() {
     isoreader.file_readers = NULL
   )
   options(default_options)
-  
+
   # register file readers
   iso_register_dual_inlet_file_reader(".did", "iso_read_did", "Isodat Dual Inlet file format (newer)", env = "isoreader")
   iso_register_dual_inlet_file_reader(".caf", "iso_read_caf", "Isodat Dual Inlet file format (older)", env = "isoreader")
-  iso_register_dual_inlet_file_reader(".di.rda", "iso_read_rda", "Isoreader R Data Archive", cacheable = FALSE, env = "isoreader")
```

Comment:
let's keep the old ones around but maybe leave a note in in the description, e.g. `Isoreader R Data Archive (deprecated)`